bin/update_sources_for_new_branch.rb

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+
+require 'bundler/setup'
+require 'manageiq/release'
+require 'trollop'
+
+opts = Trollop.options do
+  opt :branch, "The new branch name", :type => :string, :required => true
+end
+
+repos = ManageIQ::Release::Repos["master"]
+review = StringIO.new
+post_review = StringIO.new
+
+repos.each do |repo|
+  puts ManageIQ::Release.header("Updating #{repo.name}")
+  Dir.chdir(repo.path) do
+    repo.fetch
+    repo.checkout(opts[:branch])
+
+    # Update the README badges
+    readme = File.read("README.md")
+    readme.gsub!("coveralls.io/repos/#{repo.github_repo}/badge.svg?branch=master", "coveralls.io/repos/#{repo.github_repo}/badge.svg?branch=#{opts[:branch]}")
+    readme.gsub!("coveralls.io/github/#{repo.github_repo}?branch=master", "coveralls.io/github/#{repo.github_repo}?branch=#{opts[:branch]}")
+    readme.gsub!("hakiri.io/github/#{repo.github_repo}/master", "hakiri.io/github/#{repo.github_repo}/#{opts[:branch]}")
+    readme.gsub!("buildstats.info/travisci/chart/#{repo.github_repo}?branch=master", "buildstats.info/travisci/chart/#{repo.github_repo}?branch=#{opts[:branch]}")
+    File.write("README.md", readme)
+
+    # Update the Gemfile for manageiq
+    if repo.name == "manageiq"
+      gemfile = File.read("Gemfile")
+      gemfile.gsub!(/^(\s*gem "manageiq-.+:branch => )"master"$/, %Q(\\1"#{opts[:branch]}"))
```

Comment:
Unfortunately it doesn't match because there is no `manageiq-` in the method.
```ruby
gem plugin_name, :git => "https://github.com/ManageIQ/#{plugin_name}", :branch => "master"
```